plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -134,6 +143,7 @@ contract RootChain {
         bytes32 root = childChain[blknum].root;
 
         require(msg.sender == txList[6 + 2 * oindex].toAddress());
+        require(msg.value >= minExitBond);
         bytes32 txHash = keccak256(txBytes);
```

Comment:
Why would anyone submit a bond bigger then the `minExitBond`? If this is the case why not just force everyone to send the minExitBond so we don't have to keep track of the bond value in `storage`?

Reply:
Fixed. Also tested that challenger receives the bond in test_challenge_exit.